diff --git a/index.js b/index.js
index 6f1e9a30ab1382eb55a131e9e51c0d8faefa23ca..a19206db057ff8b97396100e5301980523ed0257 100644
--- a/index.js
+++ b/index.js
@@ -75,7 +75,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-android-arm64')
+        return require('node-nowplaying-android-arm64')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -87,7 +87,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-android-arm-eabi')
+        return require('node-nowplaying-android-arm-eabi')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -103,7 +103,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-win32-x64-msvc')
+        return require('node-nowplaying-win32-x64-msvc')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -115,7 +115,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-win32-ia32-msvc')
+        return require('node-nowplaying-win32-ia32-msvc')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -127,7 +127,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-win32-arm64-msvc')
+        return require('node-nowplaying-win32-arm64-msvc')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -142,7 +142,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-darwin-universal')
+        return require('node-nowplaying-darwin-universal')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -154,7 +154,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-darwin-x64')
+        return require('node-nowplaying-darwin-x64')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -166,7 +166,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-darwin-arm64')
+        return require('node-nowplaying-darwin-arm64')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -182,7 +182,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-freebsd-x64')
+        return require('node-nowplaying-freebsd-x64')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -194,7 +194,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-freebsd-arm64')
+        return require('node-nowplaying-freebsd-arm64')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -211,7 +211,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-x64-musl')
+        return require('node-nowplaying-linux-x64-musl')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -223,7 +223,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-x64-gnu')
+        return require('node-nowplaying-linux-x64-gnu')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -237,7 +237,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-arm64-musl')
+        return require('node-nowplaying-linux-arm64-musl')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -249,7 +249,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-arm64-gnu')
+        return require('node-nowplaying-linux-arm64-gnu')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -263,7 +263,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-arm-musleabihf')
+        return require('node-nowplaying-linux-arm-musleabihf')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -275,7 +275,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-arm-gnueabihf')
+        return require('node-nowplaying-linux-arm-gnueabihf')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -289,7 +289,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-riscv64-musl')
+        return require('node-nowplaying-linux-riscv64-musl')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -301,7 +301,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-riscv64-gnu')
+        return require('node-nowplaying-linux-riscv64-gnu')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -314,7 +314,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-ppc64-gnu')
+        return require('node-nowplaying-linux-ppc64-gnu')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -326,7 +326,7 @@ function requireNative() {
         loadErrors.push(e)
       }
       try {
-        return require('nowplaying-linux-s390x-gnu')
+        return require('node-nowplaying-linux-s390x-gnu')
       } catch (e) {
         loadErrors.push(e)
       }
@@ -351,7 +351,7 @@ if (!nativeBinding || process.env.NAPI_RS_FORCE_WASI) {
   }
   if (!nativeBinding) {
     try {
-      nativeBinding = require('nowplaying-wasm32-wasi')
+      nativeBinding = require('node-nowplaying-wasm32-wasi')
     } catch (err) {
       if (process.env.NAPI_RS_FORCE_WASI) {
         loadErrors.push(err)
